server/routes/prediscovery/routes.py

```
@@ -36,6 +36,44 @@ def trigger_prediscovery(user_id):
+                      AND ui_state->'triggerMetadata'->>'source' = 'prediscovery'
+                """, (org_id, user_id))
+                rows = cur.fetchall()
+
+        if rows:
+            from celery_config import celery_app
+            from chat.background.task import _update_session_status
+            for session_id, task_id in rows:
+                if task_id:
+                    try:
+                        celery_app.control.revoke(task_id, terminate=True, signal="SIGTERM")
+                    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)